### DISCUSSION TITLE
Phone number validation

### DISCUSSION THREAD

**Discussion by ygrenzinger** [goveo/react-international-phone#13] [Ideas]
Hello,

First thanks for this library which is really cool (and working this my tests contrary to some others).

So far, i don't see any way to confirm if a phone is valid or not.
But I have the feeling we can do something with the "mask" data for each country.

what do you think ?  how can I help you ? :)

**Comment by ybrusentsov** [comment]
Hey @ygrenzinger, thank you for the feedback!

Phone validation is one of the next planned features.
I consider adding something like the `usePhoneValidation` hook or the `isPhoneValid` function that will take country code and phone value as arguments.

**Reply by ybrusentsov** [reply]
@ygrenzinger 
Phone validation is implemented and ready to use 🔥 
https://react-international-phone-docs.vercel.app/docs/Usage/PhoneValidation

Would be glad to get any feedback, especially if something is missing :)

**Reply by ygrenzinger** [reply]
I didn't get back to you. We are using it and it's working like a charm. 
Thanks a lot !
Hope everything is fine for you in Kyiv

**Reply by ybrusentsov** [reply]
Awesome, glad to hear that. Thanks!

**Comment by ygrenzinger** [comment]
Hello,

Tested and it works as expected !

Thanks a lot!
Hope you are safe in Ukraine.

Le sam. 31 déc. 2022 à 00:17, Yurii Brusentsov ***@***.***> a
écrit :

> @ygrenzinger <https://github.com/ygrenzinger>
> Phone validation is implemented and ready to use 🔥
>
> https://react-international-phone-docs.vercel.app/docs/Usage/PhoneValidation
>
> Would be glad to get any feedback, especially if something is missing :)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/goveo/react-international-phone/discussions/13#discussioncomment-4558320>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAJ3BAF7WFJXJ7CY4AJSBVDWP5UPPANCNFSM6AAAAAATGRK4WU>
> .
> You are receiving this because you were mentioned.Message ID:
> <goveo/react-international-phone/repo-discussions/13/comments/4558320@
> github.com>
>


-- 
Yannick Grenzinger
